web_widget_slick/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2016-2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Slick Carousel Widget",
+    "summary": "Adds SlickJS slider widget for use as a carousel on Many2one"
+    " attachment fields in backend form views.",
+    "version": "11.0.1.0.0",
+    "category": "Web",
+    "website": "https://github.com/OCA/web",
+    "author": "LasLabs, Meet Dholakia, Odoo Community Association (OCA)",
```

Comment:
if this is just a bare migration listing yourself in CONTRIBUTORS is enough :wink: